### ISSUE TITLE
Umbenennung/ Aufteilung des Repos

### ISSUE THREAD

**Issue by ROYALBEFF** [foss-ag/workshop_minecraft-python#25]
Da wir mittlerweile ja durchaus mehr als nur Minecraft-Aufgabe in diesem Repo haben, halte ich es für angemessen das Repo entweder umzubenennen oder in mehrere Repos aufzuteilen. Wie sieht der Rest von euch das?

**Comment by dragetd** [comment]
Ja, finde ich sehr sinnvoll. Evtl. kann man sogar eine eigene 'Kategorie' erstellen.

So ein paar Random Vorschläge:
youngcoding_<projectname>
dawareness_<projectname>
school_<projectname>
atschool_<projectname>
teach_<projectname>
teaching_<projectname>

oder wir belassen es bei 'workshop'. :D

**Comment by ROYALBEFF** [comment]
Also du bist eher der Meinung, dass aufteilen besser wäre als umbenennen?

**Comment by dragetd** [comment]
Okay… vielleicht ist es doch sinnvoller, nicht zu viele Repos zu machen, weil die schnell unübersichtlich werden.

Nein, Umbennenen finde ich definitiv besser als Aufteilen!

Da reicht evtl. 'workschol_schoolprojects' oder soetwas…?

Übrigens: Wenn man Repos umbenennt, dann richtet GitHub einen 'versteckten Redirect' ein und unter der alten URL kann man immer noch clonen/pushen. Allerdings nur solange bis jemand den selben Namen wieder verwendet. GitHub passt allerdings nicht die Links (in Kommentaren oder so) auf der Repo an. Trotzdem: Umbennen ist nicht so schmerzhaft.

**Comment by ROYALBEFF** [comment]
Klingt super. Wenn es die naechsten Tage keine Einwaende gibt, wuerde ich das einfach mal machen :)
